src/games/strategy/sound/ClipPlayer.java

```diff
@@ -259,6 +258,7 @@ public static void play(String clipPath, PlayerID playerId) {
     getInstance().playClip(clipPath, playerId);
   }
 
+  private MediaPlayer mediaPlayer;//prevents the garbage collector from removing this object
```

Comment:
There seems to be some bug, that the sound stops before finished, when closing and reopening the map window. This fixed it.